### ISSUE TITLE
createInvite recycling links instead of creating new ones

### ISSUE THREAD

**Issue by Specter1337** [hydrabolt/discord.js#1824]
**Please describe the problem you are having in as much detail as possible:**
I wanted to have my bot create single-use invite links to my server to a specific channel. 
If I go to Server Settings -> Invites, and there are no valid invite links for this channel, the bot does what it's supposed to and creates an invite link for the channel, specified with an options object. However, if I tell the bot to create another invite link, while the first one is still valid/active, it will _not_ create a new invite link and it will just recycle and send the existing one instead. 

I'm not quite sure whether this is intentional or not. While this behavior might be desirable in some situations, it certainly isn't in this one. I need the bot to create unique single-use codes every time the corresponding command is called and then send them to whichever server member that's requesting an invite for their friend. 

**Include a reproducible code sample here, if possible:**
```js
var options = {
  maxAge: 3600,
  maxUses: 1
};
if (user != undefined)
  var invite = rulesChannel.createInvite(options).then(function(newInvite){
    user.send("https://discord.gg/" + newInvite.code)
    });
```

**Further details:**

- discord.js version: 11.1.0
- node.js version: 6.11.2
- Operating system: Win10 x64
- Priority this issue should have – please be realistic and elaborate if possible:
Medium, I would say. I can't really use the createInvite method if it cannot reliably create unique invite codes every time I need one.

**Comment by iCrawl** [comment]
You can pass a third option called `unique` and set the boolean to true to not recycle links.

**Comment by iCrawl** [comment]
Oh, this is actually only partially true. We didn't backport that feature to 11.1-dev yet.

Currently its only possible to do that in master, which is probably not desirable for you to switch to.
I'll get this on the backport list.

**Comment by iCrawl** [comment]
Fixed in 1fe201ae
